package/MDAnalysis/visualization/streamlines.py

```diff
@@ -61,8 +63,7 @@
 import MDAnalysis
 
 
-
-def produce_grid(tuple_of_limits, grid_spacing):
+def produce_grid(tuple_of_limits: Tuple, grid_spacing: float) -> NDArray:
```

Comment:
I think here you might want to use `Sequence` instead of `Tuple`. I used to use `Tuple` myself in these situations (arguments that accept tuples) until a student pointed out this [from the docs](https://mypy.readthedocs.io/en/stable/kinds_of_types.html):

> Usually it’s a better idea to use Sequence[T] instead of tuple[T, ...], as [Sequence](https://docs.python.org/3/library/typing.html#typing.Sequence) is also compatible with lists and other non-tuple sequences.

I believe that should be safe here since other sequences would also support the unpacking safely?